inc/notificationtargetcommonitilobject.class.php

```diff
@@ -1522,6 +1522,49 @@ function getDataForObject(CommonDBTM $item, array $options, $simple = false) {
          }
 
          $data["##$objettype.numberoftasks##"] = count($data['tasks']);
+
+         $data['timelineitems'] = [];
+
+         $options = [
+            'with_documents' => false,
+            'with_validations' => false,
+            'expose_private' => $show_private,
+         ];
+         $tmptimelineitems = [];
```

Comment:
`$tmptimelineitems` is used to temporarly store current item in loop. It should be declared in loop (an maybe renamed to remove its plural form [its ending `s`], as it it used to store only one item)

Reply:
done